.travis.yml

```diff
@@ -1,34 +1,33 @@
-dist: trusty
+dist: xenial
```

Comment:
This was updated to see if it helped with the memory issue since TravisCI does not start services by default on the newer OS. It didn't make the memory issue disappear, but the update is helpful anyways.